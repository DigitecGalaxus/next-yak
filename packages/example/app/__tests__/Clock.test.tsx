/// <reference types="jest" />
import React from "react";
import { render } from "@testing-library/react";
import { Clock } from "../Clock";

test("renders Clock component", () => {
  const { asFragment } = render(<Clock />);
  expect(asFragment()).toMatchSnapshot();
});
