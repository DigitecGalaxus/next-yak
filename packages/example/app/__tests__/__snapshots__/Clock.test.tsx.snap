// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Clock component 1`] = `
<DocumentFragment>
  <div
    class=""
  >
    <div
      class=""
    >
      <div
        class=""
      />
      <div
        class=""
        index="0"
      >
        12
      </div>
      <div
        class=""
        index="1"
      >
        1
      </div>
      <div
        class=""
        index="2"
      >
        2
      </div>
      <div
        class=""
        index="3"
      >
        3
      </div>
      <div
        class=""
        index="4"
      >
        4
      </div>
      <div
        class=""
        index="5"
      >
        5
      </div>
      <div
        class=""
        index="6"
      >
        6
      </div>
      <div
        class=""
        index="7"
      >
        7
      </div>
      <div
        class=""
        index="8"
      >
        8
      </div>
      <div
        class=""
        index="9"
      >
        9
      </div>
      <div
        class=""
        index="10"
      >
        10
      </div>
      <div
        class=""
        index="11"
      >
        11
      </div>
      <div
        class=""
      />
      <div
        class=""
      />
      <div
        class=""
      />
    </div>
  </div>
</DocumentFragment>
`;
